spec/jobs/reports/sp_issuer_user_counts_report_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Reports::SpIssuerUserCountsReport do
+  subject { described_class.new }
+
+  let(:issuer) { 'example_sp' }
+  let(:emails) {['example1@email.com', 'example2@email.com']}
+  let(:name) { 'sp-issuer-report' }
```

Comment:
what's the purpose of the `name` property? It looks like each issuer + email gets a different name? But the report name is always the same, what's the purpose of making this configurable?

Reply:
I made this a PR too soon, sorry about that. I thought it was further along than it actually was.